core/scheduler/build.gradle

```diff
@@ -66,6 +66,7 @@ dependencies {
         resolutionStrategy.force "com.typesafe.akka:akka-http-spray-json_${gradle.scala.depVersion}:${gradle.akka_http.version}"
         resolutionStrategy.force "com.typesafe.akka:akka-parsing_${gradle.scala.depVersion}:${gradle.akka_http.version}"
         resolutionStrategy.force "com.typesafe.akka:akka-http_${gradle.scala.depVersion}:${gradle.akka_http.version}"
+        resolutionStrategy.force "com.google.protobuf:protoc:${gradle.protoc.version}"
```

Comment:
I don't understand how the project can be built if a dependency is missing 🤷‍♂️

Reply:
I honestly don't understand that either.
I wonder if it's something related to to Operating System where the build is running on.
The error you get when removing that dependency is the following:
```
> Task :core:scheduler:generateProto FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':core:scheduler:generateProto'.
> Could not resolve all files for configuration ':core:scheduler:protobufToolsLocator_protoc'.
   > Could not find protoc-3.4.0-osx-aarch_64.exe (com.google.protobuf:protoc:3.4.0).
     Searched in the following locations:
         https://repo.maven.apache.org/maven2/com/google/protobuf/protoc/3.4.0/protoc-3.4.0-osx-aarch_64.exe
```
Not sure what to do to be honest.

Reply:
Alright, now makes sense, there are some `osx` compatible candidates in there with different _arch_:
https://repo.maven.apache.org/maven2/com/google/protobuf/protoc/3.4.0/

This one should probably be handled first with an issue

Reply:
I see, trying building the project within a Linux container the problem persists. 
The dependency seems needed regardless the actual operating system. I'm not fully getting it to be honest.
Shall I open already the issue on the original project? Should I wait for the final review?

Reply:
I think this problem appeared with M1 architecture. You can probably offer a solution within your PR with inspiration taken from [there](https://github.com/grpc/grpc-java/issues/7690)

Reply:
Or I can upgrade the `protoc` library, as already did, to a version >= `3.18.0` that include a version for the new architectures.
Which one do you reckon would be better?

Reply:
If a straight update solves it, I'd rather take this approach